rtil/src/threads/ue/rebo/rebo_init.rs

```diff
@@ -1388,13 +1388,9 @@ fn set_screen_percentage(percentage: f32) {
 }
 #[rebo::function("Tas::set_reticle_width")]
 fn set_reticle_width(width: f32) {
-    STATE.lock().unwrap().as_mut().unwrap().reticle_w = width;
+    RETICLE_W.store(width, Ordering::Relaxed);
```

Comment:
Instead of exposing and writing directly to the statics, setting this should be a function on `AMyHud`.